src/Translations.js

```diff
@@ -254,6 +259,9 @@ export const es = {
         "Body": "Si tiene preguntas o comentarios, por favor contactenos en ",
         "Email": "adeairquality@gmail.com"
     },
+    "Feedback" : {
+        "Title": "Please give us feedback."
```

Comment:
Is it worth trying to get a quick translation on this? Even if merged separately might be something we can do kind of quickly